### PR TITLE
Cli 52 use active profile

### PR DESCRIPTION
**Pull request by DennyGD**
Use the active profile (if set) if no profile name is supplied for `kinvey profile delete`

### CODE REVIEW

File: __snapshots__/profile-delete.test.js [hunk 1]
```diff
@@ -44,27 +44,18 @@ exports['profile delete by non-existent name when none should return error 1'] =
...
-exports['profile delete without a name should fail 1'] = `
-kinvey profile delete <name>
-
-Delete profile by name
-
-Positionals:
-  name  Profile name                                                  [required]
+exports['profile delete without a name when no active should fail 1'] = `
+[debug] Checking for package updates
+[error] ItemNotSpecified: No profile identifier is specified and/or active profile is not set.
```
Comment (dokov) [inline __snapshots__/profile-delete.test.js]: Isn't this just "and" not "and/or"?

File: lib/profile/profilesRouter.js [hunk 2]
```diff
@@ -49,8 +49,8 @@ const cmdDefinitions = [
     handlerName: 'use'
   },
   {
-    command: 'delete <name>',
-    desc: 'Delete profile by name',
+    command: 'delete [name]',
+    desc: 'Delete a specified profile or the active one',
```
Comment (dokov) [inline lib/profile/profilesRouter.js]: How about: "... the active one if no profile is specified"

File: __snapshots__/profile-delete.test.js (hunk 1 above)
Comment (DennyGD) [reply __snapshots__/profile-delete.test.js]: It is, indeed 😄

File: lib/profile/profilesRouter.js (hunk 2 above)
Comment (DennyGD) [reply lib/profile/profilesRouter.js]: I like it better like this. It's not focused so much on the active profile and it just gives you all the options. But it's up to you?

Comment (dokov) [reply lib/profile/profilesRouter.js]: OK, leave it like that then.

**Pull request merged**